src/electron/main.ts

```diff
@@ -8,28 +8,21 @@ let mainWindow: BrowserWindow;
 let ipcMainProxy: IpcMainProxy;
 
 function createWindow() {
-    // and load the index.html of the app.
-
     const windowOptions: BrowserWindowConstructorOptions = {
         width: 1024,
         height: 768,
     };
-    // Create the browser window.
 
+    const staticUrl = process.env.ELECTRON_START_URL || `file:///${__dirname}/index.html`;
```

Comment:
Part 3 - With other changes in place we can now simplify our startup to always use `loadURL(...)`

Reply:
So if webpack isn't mocking __dirname, where is it set?

Reply:
`__dirname` is a special node variable that gives you the current directory path of the currently executing code.